projects/1.16.1/assets/a-v-a-in-minecraft/ava/lang/zh_cn.json

```
@@ -1,3 +1,185 @@
+    "ava.uav_captures": "望远镜抓取",
+    "ava.uav_captured": "你被敌人的UAV发現了!",
+    "ava.ammo_kit_supplier_consume": "弹药补充器使用",
+    "ava.block_boosts_player": "玩家提升",
+    "ava.m4a1_shoot": "M4A1 开火",
+    "ava.m4a1_reload": "M4A1 更换弹药",
+    "ava.mosin_nagant_shoot": "Mosin-Nagant 开火",
+    "ava.mosin_nagant_reload": "Mosin-Nagant 更换弹药",
+    "ava.p226_shoot": "P226 开火",
+    "ava.p226_reload": "P226 更换弹药",
+    "ava.x95r_shoot": "X95R 开火",
+    "ava.x95r_reload": "X95R 更换弹药",
+    "ava.mk20_shoot": "MK20 开火",
+    "ava.mk20_reload": "MK20 更换弹药",
+    "ava.m24_shoot": "M24 开火",
+    "ava.m24_reload": "M24 更换弹药",
+    "ava.remington870_shoot": "Remington870 开火",
+    "ava.remington870_reload": "Remington870 更换弹药",
+    "ava.fn_fnc_shoot": "FN-FNC 开火",
+    "ava.fn_fnc_reload": "FN-FNC 更换弹药",
+    "ava.xm8_shoot": "XM8 开火",
+    "ava.xm8_reload": "XM8 更换弹药",
+    "ava.mp5sd5_shoot": "MP5SD5 开火",
+    "ava.mp5sd5_reload": "MP5SD5 更换弹药",
+    "ava.mk18_shoot": "MK18 开火",
+    "ava.mk18_reload": "MK18 更换弹药",
+    "ava.sr_25_shoot": "SR-25 开火",
+    "ava.sr_25_reload": "SR-25 更换弹药",
+    "ava.fg42_shoot": "FG42 开火",
+    "ava.fg42_reload": "FG42 更换弹药",
+    "ava.mp5k_shoot": "MP5K 开火",
+    "ava.mp5k_reload": "MP5K 更换弹药",
+    "ava.m202_shoot": "M202 火箭发射器 开火",
+    "ava.m202_reload": "M202 火箭发射器 更换弹药",
+    "ava.gm94_shoot": "GM-94 榴弹发射器 开火",
+    "ava.gm94_reload": "GM-94 榴弹发射器 更换弹药",
+    "ava.sw1911_shoot": "SW1911 开火",
+    "ava.sw1911_reload": "SW1911 更换弹药",
+    "ava.python357_shoot": "Python.357 开火",
+    "ava.python357_reload": "Python.357 更换弹药",
+  
+    "death.attack.ava.killed": "%1$s 被杀了",
+    "death.attack.ava.killed.weapon": "%1$s 被 %3$s 杀了",
+    "death.attack.ava.bullet": "%1$s 被 %2$s 射杀了",
+    "death.attack.ava.bullet.weapon": "%1$s 被 %2$s 用 %3$s 射杀了",
+    "death.attack.ava.explosion": "%1$s 被 %2$s 炸飞了",
+    "death.attack.ava.explosion.weapon": "%1$s 被 %2$s 用 %3$s 炸飞了",
+    "death.attack.ava.toxic_gas": "%1$s 被 %2$s 杀了",
+    "death.attack.ava.toxic_gas.weapon": "%1$s 被 %2$s 用 %3$s 杀了",
+  
+    "ava.armour.full_equipped": "当整套装备時:",
+    "ava.armour.sneak_speed_boost": "增加移动速度",
+    "ava.armour.uav_warning": "当你被敌人UAV标记時发出警告",
+    "ava.armour.night_vision": "夜视仪 N鍵 开启/关闭",
+    "ava.armour.knockback_resistance": "增加你的击退抗性",
+    "ava.armour.hurt_indicator": "伤害指示",
+  
+    "ava.gui.tab.snipers": "狙击",
+    "ava.gui.tab.rifles": "步枪",
+    "ava.gui.tab.submachine_guns": "冲锋枪",
+    "ava.gui.tab.pistols": "手枪",
+    "ava.gui.tab.miscs": "其他",
+    "ava.gui.gun_damage": "伤害",
+    "ava.gui.gun_range": "范围",
+    "ava.gui.gun_stability": "稳定性",
+    "ava.gui.gun_accuracy": "准确度",
+    "ava.gui.gun_attack_speed": "射速",
+    "ava.gui.gun_capacity": "容量",
+    "ava.gui.description.mk3a2_1": "有伤害的",
+    "ava.gui.description.mk3a2_2": "闪光弹",
+    "ava.gui.description.standard_armour_1": "提供与锁链",
+    "ava.gui.description.standard_armour_2": "盔甲同样的",
+    "ava.gui.description.standard_armour_3": "盔甲強度",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
    "ava.gui.description.standard_armour_3": "盔甲强度",
```